sdk/identity/Azure.Identity/tests/SessionRecords/ManagedIdentityCredentialImdsLiveTests/ValidateImdsUserAssignedIdentityAsync.json

```diff
@@ -50,7 +50,7 @@
         "x-ms-client-request-id": "9431caf3d5850d7ce4c6f84a60edc12f",
         "x-ms-return-client-request-id": "true"
       },
-      "RequestBody": "U2FuaXRpemVk",
```

Comment:
These bodies were base64 encoded "Sanitized". We were unconditionally setting this in the IdentityRecordedTestSanitizer even when though the actual body was null. I switched back to null due to an issue with sanitizing null requests in the Test Proxy.